sys/net/application_layer/nanocoap/Makefile.include

```diff
@@ -0,0 +1,3 @@
+ifneq (,$(filter nanocoap_dtls,$(USEMODULE)))
+  DTLS_MAX_BUF ?= 64 + 36 # default blockwise transfer size + DTLS extra bytes
```

Comment:
```suggestion
  CONFIG_NANOCOAP_BLOCKSIZE_DEFAULT := $(or $(CONFIG_NANOCOAP_BLOCKSIZE_DEFAULT),2)
  DTLS_MAX_BUF ?= "(1 << $(CONFIG_NANOCOAP_BLOCKSIZE_DEFAULT) + 3) + 36"
```

does this work?

Reply:
See https://github.com/RIOT-OS/RIOT/pull/19892/commits/5d586576e6135c9e1ac875e4f470ffafb0492449